setup.py

```diff
@@ -22,7 +22,10 @@
 import sys
 
 if sys.version_info <= (2, 6):
```

Comment:
oh, this actually does the right thing -- the version tuple for `2.6` will always have a third component.

that said, i agree it reads awkwardly, so i'm tweaking.